core/src/test/scala/unit/kafka/zk/ZkMigrationClientTest.scala

```diff
@@ -433,7 +433,7 @@ class ZkMigrationClientTest extends QuorumTestHarness {
   def migrateAclsAndVerify(authorizer: AclAuthorizer, acls: Seq[AclBinding]): Unit = {
     authorizer.createAcls(null, acls.asJava)
     val batches = new ArrayBuffer[mutable.Buffer[ApiMessageAndVersion]]()
-    migrationClient.migrateAcls(batch => batches.addOne(batch.asScala))
+    migrationClient.migrateAcls(batch => batches ++ batch.asScala)
```

Comment:
```suggestion
    migrationClient.migrateAcls(batch => batches.append(batch.asScala))
```

`++` is basically creating a new buffer here rather than adding to `batches`

Reply:
I didn't see append before, but let's try it!